examples/hubtype_demo/src/routes.ts

```diff
@@ -0,0 +1,56 @@
+import { Input, Session } from '@botonic/core'
+import DateSelected from './actions/dateSelected'
+import EmailResponse from './actions/emailResponse'
+import Greetings from './actions/greetings'
+import LogInResponse from './actions/logInResponse'
+import PaymentResponse from './actions/paymentResponse'
+import Webviews from './actions/webviews'
+import SelectCalendar from './actions/selectCalendar'
+import ResponseCalendar from './actions/responseCalendar'
+import { BankRoutes } from './routes_bank'
+
+export function routes(_: { input: Input; session: Session }): any {
+  if (
+    _.input.data == 'reset' ||
+    _.input.data == 'Reset' ||
+    _.input.data == 'hola' ||
+    _.input.data == 'Hola' ||
+    _.input.data == 'buenas' ||
+    _.input.data == 'Buenas' ||
+    _.input.data == 'hello' ||
+    _.input.data == 'Hello' ||
+    _.input.data == 'hi' ||
+    _.input.data == 'Hi' ||
```

Comment:
here we could store these keywords inside a variable holding the regex itself.
`let introRegex = /reset|hola|buenas|hello|hi/i`, with `i` flag we can ignore case.
`_input.data.match(introRegex)`.
I think that in this way would be tidier.